cylc/flow/scripts/lint.py

```diff
@@ -0,0 +1,575 @@
+#!/usr/bin/env python3
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Cylc 728 looks through one or more folders for ``suite*.rc`` files and
+search for Cylc 7 syntax which may be problematic at Cylc 8.
+
+Can be run either as a linter or "in place" (``-i``), leaving comments
+in files. If used in the "in place" mode it is recommended that you ensure
+that you have recorded the state of your workflow in a version control
+system before starting.
+
+.. warning::
+
+   When run with ``-i`` (``--inplace``) mode ``Cylc 728`` changes your files.
+   We strongly recommend committing your workflow to version control
+   before using ``Cylc 728 -i``.
+
+Usage
+^^^^^
+
+.. code-block:: bash
+
+   # run as a linter
+   cylc 728 <paths to workflow directories to check>
+
+   # run inplace
+   cylc 728 --inplace <paths to workflow directories to check>
+   cylc 728 -i <paths to workflow directories to check>
+
+   # Get information about errors:
+   cylc 728 --reference
+   cylc 728 -r
```

Comment:
This option was removed
```suggestion
```

Reply:
Yes, I don't know what happened to my branch. Working locally.